ios-contact-manager/ios-contact-manager/Controller/TableViewController.swift

```diff
@@ -0,0 +1,61 @@
+import UIKit
+
+final class TableViewController: UITableViewController {
+    private var contactsModel: ContactsModel!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configure()
+    }
+    
+    private func configure() {
+        self.contactsModel = ContactsModel()
+    }
```

Comment:
```suggestion
        self.configure()
    }
    
    private func configure() {
        self.contactsModel = ContactsModel()
    }
```
=> self 를 붙이기로 스타일 가이드를 잡으셨으면 일관되게 모두 붙여주시는게 좋습니다 ㅎㅎ